app/src/main/res/values/strings.xml

```diff
@@ -174,10 +173,10 @@
     <string name="request_timeout">Request Timeout</string>
     <string name="superuser_notification">Superuser Notification</string>
     <string name="settings_su_reauth_title">Reauthenticate after upgrade</string>
-    <string name="settings_su_reauth_summary">Reauthenticate superuser permissions after an application upgrades</string>
-    <string name="settings_su_tapjack_title">Enable Tapjacking Protection</string>
+    <string name="settings_su_reauth_summary">Ask for superuser permissions again after upgrading apps</string>
```

Comment:
Good?

Reply:
You're changing more than you have to again, you should be trying to fix things with as few changes to the original as possible.

Reply:
Yes, but saying "re-authenticate" twice isn't doing more work the second time.
Shorter, consistent, and understandable. Good qualities in my book.